examples/test_dqgmres.jl

```diff
@@ -0,0 +1,51 @@
+using Krylov, LinearOperators, ILU0
```

Comment:
```suggestion
using Krylov, LinearOperators, ILUZero
```

Package rename. See https://github.com/mcovalt/ILUZero.jl/commit/c4de1f0b5d405d917a13d85c75073f939429000a